configure.ac

```diff
@@ -39,6 +39,16 @@ else
     pkg_config_user_action="check the PKG_CONFIG_PATH environment variable"
 fi
 
+if test "x$PKG_CONFIG_PATH" = "x" && test "x$prefix" != "xNONE" && test "$host" != "$build"; then
```

Comment:
Is prefix really set to a literal "NONE"?

Reply:
yes, that's a copy-paste from further below. it works.